docs/source/asr/data/asr_results.csv

```diff
@@ -1,7 +1,15 @@
 Model Name,Model Base Class,Description
 QuartzNet15x5Base-En,EncDecCTCModel,"QuartzNet15x5 model trained on six datasets: LibriSpeech, Mozilla Common Voice (validated clips from en_1488h_2019-12-10), WSJ, Fisher, Switchboard, and NSC Singapore English. It was trained with Apex/Amp optimization level O1 for 600 epochs. The model achieves a WER of 3.79% on LibriSpeech dev-clean, and a WER of 10.05% on dev-other."
-QuartzNet15x5Base-Zh,EncDecCTCModel,"QuartzNet15x5 model trained on ai-shell2 Mandarin Chinese dataset."
-QuartzNet5x5LS,EncDecCTCModel,"QuartzNet5x5 model trained on LibriSpeech dataset only. The model achieves a WER of 5.37% on LibriSpeech dev-clean, and a WER of 15.69% on dev-other."
-QuartzNet15x5NR-En,EncDecCTCModel,"Quartznet15x5 model trained for presence of noise. The base model QuartzNet15x5Base-En was finetuned with RIR and noise augmentation to make it more robust to noise. This model should be preferred for noisy speech transcription. This model achieves a WER of 3.96% on LibriSpeech dev-clean and a WER of 10.14% on dev-other."
-Jasper10x5Dr-En,EncDecCTCModel,"JasperNet10x5Dr model trained on six datasets: LibriSpeech, Mozilla Common Voice (validated clips from en_1488h_2019-12-10), WSJ, Fisher, Switchboard, and NSC Singapore English. It was trained with Apex/Amp optimization level O1. The model achieves a WER of 3.37% on LibriSpeech dev-clean, 9.81% on dev-other."
-
+stt_en_jasper10x5dr,EncDecCTCModel,"For details about this model, please visit https://ngc.nvidia.com/catalog/models/nvidia:nemo:stt_en_jasper10x5dr"
```

Comment:
How about having some more columns like language, model, Task, etc instead of just having the name. Also, we cna drop the "For details about this model, please visit", and just put a note at the top of after table that more details can be found there.
Do we want to make QuartzNet have the same description as others?

Reply:
Done